src/Components/ArticleList.jsx

```diff
@@ -28,7 +28,7 @@ const ArticleList = () => {
         })
     }, [topic, searchParams])
 
-    if (err) return <h1>404: topic Not found </h1>
+    if (err) return <h1>404: Topic Not found </h1>
```

Comment:
What if the error is something else? Like a server error? It would be nice to either keep this a bit more generic, or use the err available to us to give the user a bit more information.